resources/pdns_authoritative_repo_debian.rb

```diff
@@ -17,53 +17,48 @@
 # limitations under the License.
 #
 
-resource_name :pdns_authoritative_install_debian
+resource_name :pdns_authoritative_repo_debian
 
-provides :pdns_authoritative_install, platform: 'ubuntu' do |node|
-  node['platform_version'].to_f >= 14.04
-end
-
-provides :pdns_authoritative_install, platform: 'debian' do |node|
-  node['platform_version'].to_i >= 8
-end
+provides :pdns_authoritative_repo, platform_family: 'debian'
```

Comment:
Are you sure you are compatible with `Debian < 8` & `Ubuntu < 14.04` ?

Reply:
The support was clearly stated in the metadata.